Bio/Data/CodonTable.py

```diff
@@ -274,18 +274,18 @@ def list_ambiguous_codons(codons, ambiguous_nucleotide_values):
     # Note ambiguous_nucleotide_values['R'] = 'AG' (etc)
     # This will generate things like 'TRR' from ['TAG', 'TGA'], which
     # we don't want to include:
-    c1_list = sorted(letter for (letter, meanings)
+    c1_list = sorted(letter for letter, meanings
                      in ambiguous_nucleotide_values.items()
-                     if set(codon[0]
-                            for codon in codons).issuperset(set(meanings)))
-    c2_list = sorted(letter for (letter, meanings)
+                     if {codon[0]
+                         for codon in codons}.issuperset(set(meanings)))
+    c2_list = sorted(letter for letter, meanings
                      in ambiguous_nucleotide_values.items()
-                     if set(codon[1]
-                            for codon in codons).issuperset(set(meanings)))
-    c3_list = sorted(letter for (letter, meanings)
+                     if {codon[1]
+                         for codon in codons}.issuperset(set(meanings)))
+    c3_list = sorted(letter for letter, meanings
                      in ambiguous_nucleotide_values.items()
-                     if set(codon[2]
-                            for codon in codons).issuperset(set(meanings)))
+                     if {codon[2]
+                         for codon in codons}.issuperset(set(meanings)))
```

Comment:
Put this and the two cases just above on one line now?

Reply:
Would put it just over 80, but would I think be more legible. Hmm, up to you - judgement time.

Reply:
It does take the lines >80, i'll leave it as it is unless you feel strongly